### PR TITLE
fix: remove additional parameters for is_paytm_activated(schema)

### LINKED ISSUE #6356: is_paytm_activated shouldn't be a read only attribute
**Describe the bug**
 Alipay, Omise, and PayPal have a property to enable them(e.g. `is_paypal_activated`). They were hybrid properties(condition was like `if client_key and secret_key then true`), so they had a dump_only=True. We need to remove this as we've moved away from such a practice. The reason for this is because we need the keys to be stored, whether the property is enabled/disabled

**Additional context**
Working on this.

### PR DESCRIPTION
**Pull request by mrsaicharan1**
Fixes #6356 
Related: https://github.com/fossasia/open-event-frontend/pull/3385
#### Short description of what this resolves:
Removes dump_only property to enable write ability to the DB.

### CODE REVIEW

**Comment by mrsaicharan1** [comment]
@iamareebjamal @uds5501 @shreyanshdwivedi Please review.

File: app/api/schema/settings.py [hunk 1]
```diff
@@ -58,7 +58,7 @@ class Meta:
     #
     # Online Payment Flags
     #
-    is_paytm_activated = fields.Bool(dump_only=True, allow_none=False, default=False)
+    is_paytm_activated = fields.Bool(allow_none=False, default=False)
```
Comment (shreyanshdwivedi) [inline app/api/schema/settings.py]: allow_none is also not required.

Comment (mrsaicharan1) [reply app/api/schema/settings.py]: Okay. Updating right now

**Comment by mrsaicharan1** [comment]
@shreyanshdwivedi @uds5501 @kushthedude PLease have a look. Removed the allow_none param too.

**Pull request merged**